model/src/main/kotlin/Identifier.kt

```diff
@@ -158,11 +156,11 @@ data class Identifier(
             "conan" -> PurlType.CONAN
             "crate" -> PurlType.CARGO
             "dep", "glide", "godep", "gomod" -> PurlType.GOLANG
```

Comment:
As you removed the mapping e.g. for `gradle` to `maven` what's the reason to keep the mapping from the go package managers to golang?
If I understand this change correctly you should also remove these as well.

If these mappings can be removed, this method can be changed to get the enum by something like
```kotlin
        companion object {
            fun from(value: String): PurlType? = values().find { it.value == value }
        }
```
and the calling method to `PurlType.from(type.toLowerCase()) ?: type.toLowerCase()`

Reply:
> As you removed the mapping e.g. for gradle to maven what's the reason to keep the mapping from the go package managers to golang?

Because AFAIK PURL does not distinguish between "godep" and "gomod", but as we emit these package types, we need to consolidate them via a mapping. But you're right that I should drop "dep" and "glide" from here as we never emit these as package types.

I'm not 100% sure how all the Golang package management works in detail. If in the end all package managers indeed on refer to Git repo with Golang code, then a future change could also change all emitted package type for Golang to "golang". But that is something I wanted to do maybe in another iteration.

> If these mappings can be removed, this method can be changed to get the enum by something like

Yes, something like this would be the goal, but I first wanted to make it obvious that this simplification would be possible and correct.